File: Semester - 01 Introduction C, C++, DSA/Introduction C++ for DSA/Module - 07.5 Practice Day/F_Equation.cpp
#include <bits/stdc++.h>
using namespace std;
int main()
{
    long long int x, n;
    cin >> x >> n;
    long long sum = 0;

    for (long long int i = 2; i <= n; i += 2)
    {
        sum += pow(x * 1.0, i * 1.0);
    }
    cout << sum;

    return 0;
}

File: Semester - 03 OOP and Python Programming/OOP and Python Programming/Week - 01 Introduction to Python/Module - 02.5 Practice Day - 01/F_Equation.py
def main():
    # tc = int(input()) 
    # for _ in range(tc):
    x, n = map(int, input().split())
    sum =0
    for i in range(2, n+1, 2):
        sum+=pow(x*1.0, i*1.0)
    print(int(sum))
if __name__ == "__main__":
    main()

